generate.py

```diff
@@ -106,14 +154,20 @@ def for_file(target_file, lnk_name=None):
     lnk.shell_item_id_list.items = elements
     return pylnk.from_segment_list(elements, lnk_name)
 
+
 def create_for_path(path, isdir):
-    entry = {}
-    entry['type'] = pylnk.TYPE_FOLDER if isdir else pylnk.TYPE_FILE
-    entry['size'] = 272896 
-    entry['created'] = datetime.now()
-    entry['accessed'] = datetime.now()
-    entry['modified'] = datetime.now()
-    entry['name'] = path.split("\\")[0]
-    return entry
-
-run()
\ No newline at end of file
+    now = datetime.now()
+    return {
+        'type': pylnk.TYPE_FOLDER if isdir else pylnk.TYPE_FILE,
+        'size': 272896,
+        'created': now,
+        'accessed': now,
+        'modified': now,
+        'name': path.split('\\')[0]
+    }
+
+
+if __name__ == '__main__':

```

Comment:
I don't think people will be loading this as a library, but whatever, I'll take it.